triggers/daml/Daml/Trigger.daml

```diff
@@ -288,7 +288,7 @@ updateImpl userTrigger msgs = do
         MCompletion completion ->
           -- NB: the commands-in-flight and ACS updateState sees are those
           -- prior to updates incurred by the msg
-          let userState = utUpdateState userTrigger.updateState state.commandsInFlight state.acs state.actAs state.readAs (MCompletion completion) state.userState
```

Comment:
I found these long lines hard to read, and they turned out to be almost identical except for the `acs` used, so I factored the common parts out into `mkUserState` below